packages/shared/src/variants/quantum.ts

```diff
@@ -234,8 +250,53 @@ export class QuantumGo extends AbstractGame<NewBadukConfig, QuantumGoState> {
     }
     throw new Error("Error finding mapped capture.");
   }
+
+
+  public bootstrap(playerB: string, playerW: string, boardSize: number, komi: number){
+
+    const currentDate: Date = new Date();
+
+    const initData: string[] = [
+
+      "(;\n", 
+      "EV[GO Variants]\n", 
+      `PB[${playerB}]\n`, 
+      `PW[${playerW}]\n`,
+      `SZ[${boardSize}]\n`,
+      `KM[${komi}]\n`,     
+      `DT[${currentDate.getMonth()+1}/${currentDate.getDate()}/${currentDate.getFullYear()} ${currentDate.getHours()}:${currentDate.getMinutes()}]\n`,
+      "VS[quantum]\n",
+      "\n",
+      "\n"
+
+    ];
+  
+    this.writeToSGF(initData, true);
+   
+  }
+  
+  //creating sgf file
+  public writeToSGF(text: string[], callingBootstrap: boolean){
+
+    for (let i = 0; i < text.length; i++){
+      let formattedMove = text[i];
+      if(!callingBootstrap){
+        formattedMove = `${i % 2 === 0 ? ";B" : ";W"}[${text[i]}]`;
+      }
+      this.sgfContent += formattedMove;
+    }
+    if(!callingBootstrap){
+      this.sgfContent += "\n\n)";
+    }  
+  }
+
+  getSGF(): string{
```

Comment:
Could we get a basic test to cover the new functionality?  QuantumGo unit tests are in [__tests__/quantum.test.ts](https://github.com/govariantsteam/govariants/blob/main/packages/shared/src/variants/__tests__/quantum.test.ts).  Something like this would be sufficient:

```
const game = new QuantumGo({
  board: { type: "grid", width: 2, height: 2 },
  komi: 5.5,
});
game.playMove(0, "aa");
game.playMove(1, "bb");
game.playMove(0, "pass");
game.playMove(1, "pass");
expect(game.getSGF()).toBe("""
 ... sgf data here ...
""")
```

Let me know if I can answer any questions about [Jest](https://jestjs.io/)

Reply:
Are there any benefits to creating a more complex test?

Reply:
More complex?  What do you have in mind?

A simple unit test is okay with me as long as this function is covered.